### PR TITLE
Update CI to Ubuntu 20 and Node 20

### PR DESCRIPTION
**Pull request by jfirebaugh**
Fixes deprecation warnings and failures in CI. See for example, https://buildkite.com/bazel/vscode-bazel-vs-bazel/builds/1993#018c22c3-eae0-4d53-96c8-7c3afe453424.

### CODE REVIEW

File: .bazelci/presubmit.yml [hunk 1]
```diff
@@ -1,9 +1,18 @@
...
-  ubuntu1804:
+  ubuntu2004:
     shell_commands:
       # Install Node.js and NPM
-      - "curl -fsSL https://deb.nodesource.com/setup_14.x | sudo -E bash - && sudo apt -y install nodejs && sudo npm install -g npm@latest"
+      # https://github.com/nodesource/distributions#installation-instructions
+      - |
+        sudo apt-get update
+        sudo apt-get install -y ca-certificates curl gnupg
+        sudo mkdir -p /etc/apt/keyrings
```
[the remark this answers is not in the document]
Comment (jfirebaugh) [reply .bazelci/presubmit.yml:10]: This isn't a GitHub Actions workflow though, it's .bazelci (Buildkite).

**Comment by alexeagle** [comment]
@meisterT FYI, John and I discussed that it might be good to add another reviewer here

**Review by meteorcloudy — APPROVED** [review]

**Comment by jfirebaugh** [comment]
I think that reviewers must be specifically authorized in repository settings somewhere in order to have their approvals unblock merging. Even with @meteorcloudy's approval I'm still getting the "Merging is blocked" message.

**Comment by meisterT** [comment]
@coeuvre is OOO this week unfortunately.

Is there another person we could include in the CODEOWNERS file to avoid reviews being blocked?

**Pull request merged**